backend/src/main/models/Project.ts

```diff
@@ -81,4 +84,10 @@ export class Project extends Model {
 
   @BelongsToMany(() => EmployeeAccount, () => WorksOn)
   employeeAccount!: EmployeeAccount[];
+
+  @HasMany(() => Task)
+  tasks!: Task[];
+  
+  @HasMany(() => Feedback)
```

Comment:
I think here it should be hasone instead of hasmany. Look at the documentation here for examples:
https://www.npmjs.com/package/sequelize-typescript

Reply:
According to the E/R, it's a one-to-one relation

Reply:
A project only has one feedback

Reply:
nevermind